docs/platform/api/nodebalancers.md

```diff
@@ -0,0 +1,166 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Using the Linode API, it is possible to set up NodeBalancers without using the Linode Manager. This guide will demonstrate the API calls needed to add and configure a NodeBalancer with two back end nodes.'
+og_description: 'Using the Linode API, it is possible to set up NodeBalancers without using the Linode Manager. This guide will demonstrate the API calls needed to add and configure a NodeBalancer with two back end nodes.'
+keywords: ["nodebalancer", "load balancing", "high availability", "linode api"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+aliases: ['linux-tools/common-commands/tail/']
+modified: 2018-04-05
+modified_by:
+  name: Linode
+published: 2018-04-05
+title: Configure NodeBalancers with the Linode API
+external_resources:
+  - '[Getting Started with NodeBalancers](https://linode.com/docs/platform/nodebalancer/getting-started-with-nodebalancers/)'
+---
+
+[NodeBalancers](https://www.linode.com/nodebalancers) can be used to provide high availability load balancing for almost any type of website or service hosted on a Linode. This guide will demonstrate how to use the Linode API to create a NodeBalancer with two back end nodes.
+
+You will need an access token for the Linode API to complete the steps in this guide. The easiest way to get an access token is by visiting the beta [Linode Manager](https://cloud.linode.com/profile/tokens); for more information, as well as a basic introduction to using the Linode API, see our guide on [Getting Started with the Linode API](/docs/platform/api/getting-started-linode-api).
+
+## Create a NodeBalancer
+
+1.  Using a text editor, create a file to store configuration options:
+
+    {{< file "nodebalancer.json" json >}}
+{
+  "region": "us-central",
+  "label": "nodebalancer-1",
+  "client_conn_throttle": 10
+}
+{{< /file >}}
+
+2.  Create a NodeBalancer by making a POST request to the `/nodebalancers` endpoint:
+
+        curl https://api.linode.com/v4/nodebalancers \
+        -H "Authorization: Bearer $TOKEN" \
+        -H "Content-Type: application/json" \
+        -X POST -d @nodebalancer.json
+
+3.  If the NodeBalancer is successfully created, the response will include its ID. Copy the ID to use in subsequent requests.
+
+## Add Configuration
+
+NodeBalancers are created without a configuration profile attached, so you will need to add one.
```

Comment:
Maybe something about how NodeBalancer configs are configuring a single port on the NodeBalancer, and that the NodeBalancer will begin listening for traffic on that port once it's configured?